README.md

```diff
@@ -141,95 +121,55 @@ This proposal is motivated by a number of cases:
+  ...
   ```
-- Non memory/IO applications:
+- Non-blocking memory/IO applications:
   ```js
-  import { ReaderWriterLock } from "prex";
+  import { ReaderWriterLock } from "...";
   const lock = new ReaderWriterLock(); 
   
   export async function readData() {
     // wait for outstanding writer and take a read lock
-    try using (await lock.read()) { 
-      ... // any number of readers
-      await ...; 
-      ... // still in read lock after `await`
-    } // release the read lock
-  }
+    using const (await lock.read());
+    ... // any number of readers
+    await ...; 
+    ... // still in read lock after `await`
+  } // release the read lock
   
   export async function writeData(data) {
     // wait for all readers and take a write lock
-    try using (await lock.write()) { 
-      ... // only one writer
-      await ...;
-      ... // still in write lock after `await`
-    } // release the write lock
-  }
+    using const (await lock.write());
+    ... // only one writer
+    await ...;
+    ... // still in write lock after `await`
+  } // release the write lock
   ```
 
 # Prior Art
 
 <!-- Links to similar concepts in existing languages, prior proposals, etc. -->
 
-- C#: [`using` statement](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-statement)  
+- C#: 
+  - [`using` statement](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/using-statement)  
+  - [`using` declaration](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-8.0/using#using-declaration)  
 - Java: [`try`-with-resources statement](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html)  
 - Python: [`with` statement](https://docs.python.org/3/reference/compound_stmts.html#the-with-statement) 
 
 # Syntax
 
-## `try using` Blocks
-
-```js
-// for a synchronously-disposed resource:
-
-// 'try' with expression resource
-try using (expr) {
-  ...
-}
-
-// 'try' with local binding
-try using (const x = expr1) {
-  ...
-}
-
-// 'try' with multiple local bindings
-try using (const x = expr1, y = expr2) {
-  ...
-}
-
-// for an asynchronously disposed resource in an async function:
-
-// 'try' with expression resource
-try using await (obj) {
-  ...
-}
-
-// 'try' with local binding
-try using await (const x = expr1) {
-  ...
-}
-
-// 'try' with multiple local bindings
-try using await (const x = expr1, y = expr2) {
-  ...
-}
-
-```
-
-## `using value` Statements and `using const` Declarations
+## `using const` Declarations
 
 ```js
 // for a synchronously-disposed resource (block scoped):
-using value expr;                         // no local binding
-using const x = expr1;                    // local binding
-using const y = expr2, z = expr3;         // multiple bindings
+using const x = expr1;                              // resource w/ local binding
+using const (expr);                                 // resource w/o local binding
```

Comment:
The mental model treat "const x" as a whole part (I know it's not in the syntax level), `using` is a modifier. So I think it's strange to have this `w/o` binding version.

Reply:
We've discussed this at prior meetings. A bindingless version is important to a number of scenarios (i.e., tracing, locking, Go `defer`-like operations, etc.) where the binding itself would otherwise be unused. If I need to go back to `using value expr;` in a separate statement, I will, but I'd rather not move forward without *some* form of bindingless `using` declaration.

Reply:
What about `using const void = expr`? I've seen that in other issues and think that is good in some way